avocado/utils/data_structures.py

```diff
@@ -243,3 +245,58 @@ def time_to_seconds(time):
     else:
         seconds = 0
     return seconds
+
+
+class DataSize(object):
+    """
+    Data Size object with builtin unit-converted attributes.
+
+    :param data: Data size plus optional unit string. i.e. '10Mb'. No
+                 unit string means the data size is in bytes.
+    """
+
+    def __init__(self, data):
+        self.__multipliers = OrderedDict([('b', 1),  # 2**0
```

Comment:
This could be moved to the class level, since it's a *constant* mapping that is reused in other methods.

Reply:
Sure.